### ISSUE TITLE
5.x POP_EST in cultural dbf files have Double datatype 

### ISSUE THREAD

**Issue by andrew-wils** [nvkelso/natural-earth-vector#846]
I noticed that starting with 5.0, the cultural DBF files declare the POP_EST field as having double precision floating-point values, instead of integers, which is unexpected for population counts.

The spreadsheet file in the housekeeping directory still formats the field as Integer.

Is this an intentional change? Are you preparing for population counts that are greater than the maximum value of the SHP integer type?

Thanks!

**Comment by nvkelso** [comment]
That's awkward, thanks for flagging it here.

I suspect this has to do with a Mapshaper config? Can you confirm one or two files you see this problem on? I'm assuming "admin_0_countries.dbf" is one of them...

**Comment by andrew-wils** [comment]
Yes, I encountered it with admin_0_countries.dbf and lake boundary variants on all levels of detail.
There are plenty of integer fields, like GDP_MD.
